server/src/actions/updateUser.ts

```diff
@@ -0,0 +1,28 @@
+import { doc, getDoc, updateDoc } from '@firebase/firestore'
+import { users } from '../utilities/firebaseInit'
+import { geohashForLocation} from 'geofire-common'
+
+export const toggleUserConnectionStatus = async (userId: string) => {
+    const userRef = doc(users, userId)
+    const userDoc = await getDoc(userRef)
+
```

Comment:
Possible error if user doesn't exist? Should there be an issue for people to make try catches for all firebase requests for safety?

Reply:
Good idea, I think this would be a good issue for the backend.